cmd/minishift/cmd/config/view.go

```diff
@@ -59,6 +59,7 @@ var configViewCmd = &cobra.Command{
 
 func init() {
 	excludedConfigKeys["addons"] = true
+	excludedConfigKeys["cache-images"] = true
```

Comment:
👍

Reply:
👎 
This, with addons, should be part of the Instance config `minishiftConfig`. In that case, this would never have been needed,